build-aux/couchdb-build-release.sh

```diff
@@ -23,6 +23,7 @@ cd src/
 
 for repo in *; do
   cd $repo
+  grep -rl '{vsn, git}' | xargs sed -i "s/{vsn, git}/{vsn, \"${VERSION}-dev\"}/"
```

Comment:
hm, I think just the value of `git describe --always --tags`. the $VERSION value should be in the tarball file name and in the GET / welcome message.